cli/cliui/table_test.go

```diff
@@ -188,6 +193,31 @@ foo   foo1        foo3              2022-08-02T15:49:10Z
 		compareTables(t, expected, out)
 	})
 
+	t.Run("Inline", func(t *testing.T) {
+		t.Parallel()
+
+		expected := `
+NAME    AGE
+Alice   25
+		`
+
+		inlineIn := []tableTest4{
+			{
+				Inline: tableTest2{
+					Name: stringWrapper{
+						str: "Alice",
+					},
+					Age:         25,
+					NotIncluded: "IgnoreMe",
+				},
+			},
+		}
+		out, err := cliui.DisplayTable(inlineIn, "", []string{"name", "age"})
+		log.Println("rendered table:\n" + out)
```

Comment:
Is this log intentional?

Reply:
Yes, it's helpful to build the expected output. Test logs only log in the error case by default (unless you do `go test -v`)